python/pyspark/context.py

```diff
@@ -534,6 +534,27 @@ def setLogLevel(self, logLevel: str) -> None:
         """
         self._jsc.setLogLevel(logLevel)
 
+    def setLogLevel(self, logName: str, logLevel: str) -> None:
```

Comment:
Yeah, I mean, this is all you've exposed in R and Python, so not sure you should/need to expose more in Java/Scala